plugins/modules/packaging/os/zypper.py

```diff
@@ -213,6 +214,7 @@
     ZYPP_LOCK_TIMEOUT: 20
 '''
 
+import pathlib
```

Comment:
`pathlib` is not available in Python 2. Please use `os.path` instead.

Reply:
Fixec